packaging/rpm/microshift.spec

```diff
@@ -205,14 +205,14 @@ install -d -m755 %{buildroot}%{_sharedstatedir}/microshift-backups
 install -d -m755 %{buildroot}%{_sysconfdir}/crio/crio.conf.d
 
 %ifarch %{arm} aarch64
-install -p -m644 packaging/crio.conf.d/microshift_arm64.conf %{buildroot}%{_sysconfdir}/crio/crio.conf.d/microshift.conf
+install -p -m644 packaging/crio.conf.d/10-microshift_arm64.conf %{buildroot}%{_sysconfdir}/crio/crio.conf.d/10-microshift.conf
 %endif
 
 %ifarch x86_64
-install -p -m644 packaging/crio.conf.d/microshift_amd64.conf %{buildroot}%{_sysconfdir}/crio/crio.conf.d/microshift.conf
+install -p -m644 packaging/crio.conf.d/10-microshift_amd64.conf %{buildroot}%{_sysconfdir}/crio/crio.conf.d/10-microshift.conf
```

Comment:
So, what happens if we have an existing package with `crio/crio.conf.d/microshift.conf` file installed and we uograde to a new one? Do we end up having both files?

Reply:
I checked that and seems that updating is actually removing old and installing new (notice cleanup is done against older `c831` version):
```
  Upgrading        : microshift-4.16.0_0.nightly_2024_01_31_073538_20240202100244_acd1e6272-1.el9.x86_64                                                           4/12
  Running scriptlet: microshift-4.16.0_0.nightly_2024_01_31_073538_20240202100244_acd1e6272-1.el9.x86_64                                                           4/12
  Running scriptlet: microshift-4.16.0_0.nightly_2024_01_31_073538_20240201204716_c8318134a-1.el9.x86_64                                                           9/12
  Cleanup          : microshift-4.16.0_0.nightly_2024_01_31_073538_20240201204716_c8318134a-1.el9.x86_64                                                           9/12
  Verifying        : microshift-4.16.0_0.nightly_2024_01_31_073538_20240202100244_acd1e6272-1.el9.x86_64                                                           1/12
  Verifying        : microshift-4.16.0_0.nightly_2024_01_31_073538_20240201204716_c8318134a-1.el9.x86_64                                                           2/12

```

Also, contents of the `etc/crio/crio.conf.d` are as expected:
```
$ ls /etc/crio/crio.conf.d/
microshift.conf  microshift-ovn.conf

# after update

$ ls /etc/crio/crio.conf.d/
10-microshift.conf  11-microshift-ovn.conf
```